### PR TITLE
Homepage navbar devise views with bootstrap

### PR DESCRIPTION
**Pull request by Tenicia**

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class PagesController < ApplicationController
+  before_action :authenticate_user!
+  def home
```
Comment (arnoFleming) [inline app/controllers/pages_controller.rb]: Put empty method definitions on a single line.

File: spec/controllers/pages_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe PagesController, type: :controller do
+
```
Comment (arnoFleming) [inline spec/controllers/pages_controller_spec.rb]: Extra empty line detected at block body beginning.

File: spec/features/navbar_spec.rb [hunk 3]
```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```
Comment (arnoFleming) [inline spec/features/navbar_spec.rb]: Block has too many lines. [37/33]

Comment (Tenicia) [inline spec/features/navbar_spec.rb]: Deleted a test

**Pull request merged**